src/consumption.cpp

```diff
@@ -14,10 +14,13 @@
 #include "mutation.h"
 #include "options.h"
 #include "translations.h"
+#include "units.h"
 
 #include <string>
 #include <algorithm>
 
+using namespace units::literals;
```

Comment:
This is how the standard library does it (`std::literals`).

The namespace only contains the user defined literals. Yes, you *can* move all the literals into the global namespace and thereby get rid of the using, but that includes them all the time, especially in headers. The `using` only includes it in the source files where it is actually needed.